packages/react-components/src/components/CaptionsBanner.tsx

```diff
@@ -94,18 +69,22 @@ export const _CaptionsBanner = (props: _CaptionsBannerProps): JSX.Element => {
           ) : (
             <Persona {...personaOptions} />
           );
+          // if display name is too long, replace with ...
+          if (caption.displayName.length > 15) {
+            caption.displayName = `${caption.displayName.substring(0, 16)}...`;
```

Comment:
Can we use css trick to do this? we should not limit how many characters, everything should depend on max length of parent container: https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow

Same amount of characters doesn't mean they have same length let's take an extreme example:

10 ‘l’:
llllllllll
10 'P':
PPPPPPPPPP